mpi-jobs.shtml

```diff
@@ -186,11 +212,14 @@ executable = <i>run_mpi</i>.sh
 # arguments = (if you want to pass any to the shell script)
 should_transfer_files = YES
 when_to_transfer_output = ON_EXIT
-transfer_input_files = <i>input_files</i>, <i>myprogram</i>
+transfer_input_files = (this should be a comma separate list of input files if needed)
 
-## Required options for accessing modules
-requirements = ( Target.HasModules == true )
 getenv = true
+# Requirement for accessing new set of modules
+requirements = ( Target.HasChtcSoftware == true ) 
+# Old requirement for previous set of MPI modules
+# These modules will not be available after April 20
+# requirements = ( Target.HasModules == true )
```

Comment:
licensed_software.shtml doesn't prefix these attributes with Target.  I'm forgetting right now which syntax is correct (or more correct), but they should be unified.

Reply:
Technically `Target.` is more correct but I don't think we need it + we don't use it for other requirements. I'll remove.